src/main/java/pl/scoutCamp/api/controller/rest/TeamCategorizationSheetRestController.java

```diff
@@ -0,0 +1,85 @@
+package pl.scoutCamp.api.controller.rest;
+
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.scoutCamp.api.dto.*;
+import pl.scoutCamp.api.dto.mapper.*;
+import pl.scoutCamp.business.*;
+import pl.scoutCamp.domain.TeamCategorizationSheet;
+import pl.scoutCamp.domain.TeamCategorizationSheetAssignment;
+
+import java.net.URI;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping(TeamCategorizationSheetRestController.API_TEAM_SHEETS)
+public class TeamCategorizationSheetRestController {
+
+    public static final String API_TEAM_SHEETS = "/categorization_team_sheet";
+    public static final String NEW_TEAM_SHEET = "/new";
+    public static final String TEAM_SHEET_RESULT = "/%s";
+
+    private TeamService teamService;
+    private CategoryService categoryService;
+    private CategorizationSheetService categorizationSheetService;
+    private TeamCategorizationSheetService teamCategorizationSheetService;
+    private TeamCategorizationSheetAssignmentService teamCategorizationSheetAssignmentService;
+    private TeamCategorizationSheetAssignmentMapper teamCategorizationSheetAssignmentMapper;
+    private TeamCategorizationSheetMapper teamCategorizationSheetMapper;
+    private CategorizationSheetMapper categorizationSheetMapper;
+    private CategoryMapper categoryMapper;
+    private TeamMapper teamMapper;
+
+    @PostMapping(value = NEW_TEAM_SHEET)
+    public ResponseEntity<TeamCategorizationSheetDTO> addTeamSheetAssignments(
+            @RequestParam int teamId,
+            @RequestParam int points,
+            @RequestParam int categoryId,
+            @RequestParam int categorizationSheetId,
+            @RequestParam(defaultValue = "true") boolean isDraft,
+            @RequestBody List<TeamCategorizationSheetAssignmentDTO> teamCategorizationSheetAssignmentsDTO
```

Comment:
I am not sure if splitting passing data on Parameters and Request Body is proper approach. Can't we create new DTO for passing all data in the Request Body?